internify/src/components/pages/CreateJobPosting/TechRequirements.js

```diff
@@ -82,6 +83,7 @@ function TechRequirements(props) {
   return (
     <div className="Create">
       <Container maxWidth="md" style={{ padding: "0 10em" }}>
+      <AuthNavBar></AuthNavBar>
```

Comment:
Use the short-hand notation I mentioned if you don't plan on adding content inside this component call